src/retrace/retrace_worker.py

```diff
@@ -725,8 +723,7 @@ def start_vmcore(self, custom_kernelver=None):
                         crash_foreach_bt = None
 
             except Exception as ex:
-                log_error(str(ex))
-                self._fail()
+                raise Exception(str(ex))
```

Comment:
Does this make sense? To catch it and then re-raise it immediately?

Reply:
As far as I understand this is the way you allow a caller to handle this exception which is what we want here, though I'm no Python expert.  Do you see an alternative or see a problem?

Reply:
Ah yes ok I see the point.  I think the right thing to do there is to have a log message and raise the exception there similar to the others.  I will resubmit.